### DISCUSSION TITLE
few-shot promptings extended

### DISCUSSION THREAD

**Discussion by koolamusic** [openai/openai-python#310] [Ideas]
Hi guys, kudos to the wonderful work being done here. 

I was going through the `ChatMarkupLangugage` guide and noticed the few shot prompting examples, however I was wondering, can Assistant hold a conversation between two or more persons? 

- Assistant
- User
- Manager

In this case Manager is superior to User and can tell assistant how to server `User` better.

**Comment by athyuttamre** [comment]
You can pass in the `name` parameter to enable this behavior (and steer it further by instructing the model in the system prompt) e.g. `{"role": "user", "name": "manager", "content": "..."}`

**Reply by koolamusic** [reply]
Thanks
